scripts/POLLING_PLACES.md

```diff
@@ -0,0 +1,20 @@
+# Polling Places helper scripts
+
+Some years during election season we enable a page to help riders search for their polling place and get transit directions to it. In order to support this functionality, we need to gather current data on polling places and district shapes each year. There are a few scripts to help automate this.
```

Comment:
Can you add a note here describing the fact that the scripts don't output 100% correct info and manual adjustment is a must?